src/stdlib_stringlist_type.f90

```diff
@@ -84,8 +84,20 @@ module stdlib_stringlist_type
                                                                 insert_before_chararray_int,    &
                                                                 insert_before_stringarray_int
 
-        procedure         :: get_string_idx         => get_string_idx_wrap
-        generic, public   :: get                    => get_string_idx
+        procedure         :: get_idx                        =>  get_idx_impl
+        procedure         :: get_range_idx                  =>  get_range_idx_impl
+        generic, public   :: get                            =>  get_idx,                        &
+                                                                get_range_idx
+        
+        procedure         :: pop_idx                        =>  pop_idx_impl
+        procedure         :: pop_range_idx                  =>  pop_range_idx_impl
+        generic, public   :: pop                            =>  pop_idx,                        &
+                                                                pop_range_idx
```

Comment:
```suggestion
        procedure         :: pop_index               ! or `pop_idx`, if you wish
        procedure         :: pop_range_index         ! or `pop_range_idx`, if you wish
        generic, public   :: pop                =>  pop_index, pop_range_index
```

In the executable statements you can use `pop_index` or `pop_range_index`.

The same can be done for the other TBPs with the `_impl` suffix.

The rationale of the (optional) short version of TBPs is to help you create shorter, more convenient names, not to create additional naming problems :wink: .

Please let me know if it's still unclear.

Reply:
Thanks, got it. Wish I had known this before 😅.